test/e2e/soak_k8petstore.go

```diff
@@ -0,0 +1,231 @@
+/*
```

Comment:
Is there any reason why this should not go in test/e2e/examples.go?
Also, prefixing the file name with "soak_" is inconsistent with all other e2e tests.